src/Uno.UI/DirectUI/CalendarConstants.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Linq;
+
+namespace DirectUI
+{
+	internal class CalendarConstants
```

Comment:
What about leap second? Shouldn't ticksPerHour, and all other constans, be 1 second longer?

Reply:
Those constants are from the window's source code. Didn't wait for a leap second to validate, but I would assume they are handle in another way. Anyway for a `Calendar` the leap seconds is not really a concern.